exercises/reverse-string/tests/reverse-string.rs

```diff
@@ -62,3 +62,11 @@ fn test_a_palindrome() {
 fn test_wide_characters() {
     process_reverse_case("子猫", "猫子");
 }
+
+#[test]
+#[ignore]
+/// bonus only
+/// grapheme clusters
+fn test_grapheme_clusters() {
+    process_reverse_case("uüu", "uüu");
```

Comment:
This isn't displaying correctly in Firefox 60.
Chrome correctly shows the strings as being symmetrical